scripts/utils/migrate/get-frontmatter.js

```diff
@@ -54,18 +54,21 @@ const addCustomFrontmatter = {
       topics,
     };
   },
-  [TYPES.BASIC_PAGE]: ({ doc, topics }, defaultFrontmatter) => {
+  [TYPES.BASIC_PAGE]: ({ doc, topics, redirects }, defaultFrontmatter) => {
+    const frontmatter = {
+      ...defaultFrontmatter,
+      topics,
+      redirect_from: redirects,
```

Comment:
Can we instead use `redirects` so that we are consistent with the [dev site](https://raw.githubusercontent.com/newrelic/developer-website/b575317acf8b2962f2c2fd1e82a64c4efd5b0e29/src/markdown-pages/automate-workflows/index.mdx)?

Reply:
I _think_ one of the plugins we were talking about using expects this format?

Reply:
yeah, i used `redirects_from` to make it compatible with that plugin, but since we're likely not going to use said plugin, i'll change it to `redirects`.